api/external/infra_proxy/infra_proxy.proto

```diff
@@ -215,6 +218,9 @@ service InfraProxy {
     option (google.api.http).get = "/api/v0/infra/servers/{server_id}/orgs/{org_id}/environments/{name}/recipes";
     option (chef.automate.api.iam.policy).resource = "infra:infraServers:{server_id}:orgs:{org_id}:environments";
     option (chef.automate.api.iam.policy).action = "infra:infraServers:get";
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "hidden";
```

Comment:
Curious what we are suppressing here...?

Reply:
Picked up files from un-stashing.